### DISCUSSION TITLE
Add international Standards Symbols + Hazard Symbols.

### DISCUSSION THREAD

**Discussion by Cogs16** [FakeFishGames/Barotrauma#6058] [Ideas and suggestions]
I think the sub editor could use more symbols, which would help with both clarity and subs designed for multi-lingual use. Here are some examples.

https://en.wikipedia.org/wiki/ISO_7010

https://en.wikipedia.org/wiki/Hazard_symbol

Additionally, making the crew insignia symbols available in the sub editor would be a big plus.

**Comment by Veritius** [comment]
Cogs, learn XML and make content packages for what you want. In my opinion, you're clogging the discussions section with fairly meaningless suggestions that don't contribute to the substance of the game.

**Reply by Singularia** [reply]
IMO it would be really useful and nice to have symbols at least for some hazards: for example signs for laser beams, explosives, radiation, electricity, fire, bio and chemical hazards. Right now we don't have anything with only exception being 2 radiation signs one of which is a label for a reactor room.
Also additional labels for some rooms would be also nice to have. For example labels for Gunnery, Berthing and Crew Compartments, O2 room, Utility and Brig.
